src/test/suite/stubDemoTutorial.test.ts

```diff
@@ -97,6 +111,82 @@ suite('stub out a tutorial', () => {
 		}
 	}
 
+	async function validateTerminalResponseWin(terminalName : string, terminalText : string, terminalResponse : string[]) {
+		console.log(`validateTerminalResponse terminal ${terminalName} executing text ${terminalText} on a Windows machine`);
+		const term = window.createTerminal(terminalName);
+		expect(term).to.not.be.null;
+		if (term) {
+			console.log(`-current terminal = ${term?.name}`);
+			await sendTerminalText(terminalName, terminalText);
+			await waitUntil(async () => {
+				await focusOnNamedTerminal(terminalName);
+				return terminalName === window.activeTerminal?.name;
+			}, 1000);
+			try {
+				const terminalData = await collectTerminalStringsAsArray();
+				let foundExpectedString = false;
+				const responseArray = terminalResponse as string[];
+				console.log(`-terminal output =`);
+				responseArray.forEach(element => {
+					console.log(`---> ${element}`);
+				});
+				let foundIt = true;
+				for (let index = 0; index < responseArray.length; index++) {
+					const response = responseArray[index];
+					if (!response.startsWith(`PS`)) { // PowerShell lines begin with PS and we just want the others
+						const foundNum = searchStringInArray(response, terminalData);
+						foundIt = foundNum > -1;
+						if (!foundIt) {
+							break;
+						}
+					}
+				}
+				foundExpectedString = foundIt;
+
+				if (foundExpectedString) {
+					return;
+				} else {
+					fail(`Searching for ${terminalResponse} but not found in current content of active terminal ${window.activeTerminal?.name} : ${terminalData}`);
+				};
+			} catch (error){
+				fail(error);
+			} finally {
+				findAndDisposeTerminal(terminalName);
+			}
+		}	
+	}
+
+	async function collectTerminalStringsAsArray() : Promise<string[]> {
+		let terminalData:string[] = [];
+		const predicate = async () => {
+			const result: string = await getActiveTerminalOutput();
+			await commands.executeCommand('workbench.action.terminal.clear');
```

Comment:
what is the purpose of clearing the terminal?

Reply:
Clearing the terminal means that we will not get duplicated material between retrievals. If we don't clear it there's a good chance we will get the same info on multiple lines and we don't want/need that.

Reply:
I do niot understand why there will be the same info several times. We are calling again the command between each clean?

The risk of clearing is that if the text is provided between the previous line when retrieving the content and this lien of clearing the test will fail as the important part would have been cleared.

Reply:
The goal here is to collect the information in the terminal. I want to avoid duplication. If I collect terminal text the first time and it has 1,2,3,4,5; and then I collect it a second time and it has 1,2,3,4,5,6,7,8 -- I can tell that there has been some additional text added to the terminal since the first time I checked. Since we are never sure how long it will take for a terminal command to finish executing, I want to collect information from the terminal a few times to ensure that I have it all or as much as I can get. I don't need to collect 1,2,3,4,5 again -- so if I clear the terminal text after I collect it, I am ensured that I will only get 6,7,8

Reply:
> so if I clear the terminal text after I collect it, I am ensured that I will only get 6,7,8

unfortunately not. You might have only 7, 8. if the 6 appears between the retrieval of the text and the clear command is executed

Reply:
No longer needed with the simplified test using double quotes around the echo text